### PR TITLE
Add to the webhook guide that we call the webhook for canceling a refund

### PR DESCRIPTION
**Pull request by ndijkstra**

### CODE REVIEW

File: source/changelog/v2/changelog.rst [hunk 1]
```diff
@@ -56,6 +56,11 @@ Monday, 25th
 
 - Updated the list of available profile :doc:`merchant category codes </reference/v2/profiles-api/create-profile>`
 
+Wednesday, 6th
```
Comment (willemstuursma) [inline source/changelog/v2/changelog.rst]: This seems to be in the wrong spot.

Comment (ndijkstra) [reply source/changelog/v2/changelog.rst]: This change is already live since February 6th

Comment (willemstuursma) [reply source/changelog/v2/changelog.rst]: Screen Shot 2019-04-30 at 14 53 10

I mean 6th is between 21st and 25th.

Comment (ndijkstra) [reply source/changelog/v2/changelog.rst]: Totally missed that 😅 should be ok now.

**Pull request merged**